From 3f947123abfce1a9f80746c95819172c6a6fae8d Mon Sep 17 00:00:00 2001
From: Colin Guthrie <colin@mageia.org>
Date: Thu, 26 Apr 2012 09:46:49 +0100
Subject: [PATCH] mageia: Correct usage of M4_DEFINES vs. M4_DISTRO_FLAG.

This seems to have been a problem since mageia support was merged,
as upstream had changed how this bit worked without us reaslising.
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 53c8f0e..b1a466c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -493,7 +493,7 @@ case $with_distro in
         mageia)
                 SYSTEM_SYSVINIT_PATH=/etc/rc.d/init.d
                 AC_DEFINE(TARGET_MAGEIA, [], [Target is Mageia])
-                M4_DISTRO_FLAG=-DTARGET_MAGEIA=1
+                M4_DEFINES=-DTARGET_MAGEIA=1
                 have_plymouth=yes
                 ;;
         other)
-- 
1.7.10

